components/arm/universalrobots/ur.go

```diff
@@ -462,7 +462,7 @@ func (ua *URArm) MoveToJointPositionRadians(ctx context.Context, radians []float
 
 		// TODO(erh): make responsive on new message
 		if !goutils.SelectContextOrWait(ctx, 10*time.Millisecond) {
-			return ctx.Err()
+			return ua.Stop(ctx, nil)
```

Comment:
Should we just return early, instead of calling Stop on the arm? How long does Stop stop the arm for? I want to make sure this will work for the future where we want to cancel the current Move request but allow a subsequent Move request to succeed.

Reply:
`ua.Stop` will not hold the robot in an unresponsive state. The command that gets issued to a UR in this driver could immediately be followed by a new Motion request (and its accompanying controller commands). There is a risk that we issue a new command before the arm has actually come to a complete stop, although that would require near instantanous planning on `motionplan`s part.

Reply:
ok, sounds good